hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/CompactionLogDagPrinter.java

```diff
@@ -40,14 +40,14 @@
 public class CompactionLogDagPrinter extends Handler
     implements SubcommandWithParent {
 
-  @CommandLine.Option(names = {"-f", "--file-name"},
-      description = "Name of the image file. (optional)")
-  private String fileName;
+  @CommandLine.Option(names = {"-f", "--file-name-prefix"},
+      description = "Prefix to be use in image file name. (optional)")
+  private String fileNamePrefix;
 
   // TODO: Change graphType to enum.
   @CommandLine.Option(names = {"-t", "--graph-type"},
-      description = "Type of node name to use in the graph image.\n" +
-          "Accepted values are: \n" +
+      description = "Type of node name to use in the graph image. " +
+          "(optional)\n Accepted values are: \n" +
           "  file_name: to use file name as node name in DAG,\n" +
```

Comment:
```suggestion
          "  file_name (default): to use file name as node name in DAG,\n" +
```

Reply:
If the default is already indicated by PicoCLI then ignore this comment.

Reply:
It doesn't apparently. Updated as suggested.